lib/nitra/worker.rb

```diff
@@ -287,6 +287,24 @@ def debug(*text)
           channel.write("command" => "debug", "text" => text.join, "on" => on)
         end
       end
+
+      def unique_unique_output_file_for(filename, template_output_file)
```

Comment:
`unique_unique_output_file_for` Is this a typo or an way to do override in Ruby?

Reply:
No, it's a typo.